common/events/player_events.h

```diff
@@ -976,6 +982,72 @@ namespace PlayerEvent {
 			);
 		}
 	};
+
+	struct ParcelRetrieve
+	{
+        uint32      item_id;
+        uint32      quantity;
+        std::string from_player_name;
+        uint32      sent_date;
+
+		// cereal
+		template<class Archive>
+		void serialize(Archive &ar)
+		{
+			ar(
+				CEREAL_NVP(item_id),
+				CEREAL_NVP(quantity),
+				CEREAL_NVP(from_player_name),
+				CEREAL_NVP(sent_date)
+			);
+		}
+	};
```

Comment:
We spoke about this, these structs have spaces versus tabs so indentation looks off here versus in editor

Reply:
Resolved